src/style.css

```diff
@@ -2,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). However, in this case, since you are styling the `body` element it's acceptable. But be mindful not to use other tag names for styling elsewhere.